<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaCadastrarVeiculo</class>
 <widget class="QDialog" name="TelaCadastrarVeiculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>406</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PLACA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPlaca">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>VALOR DI√ÅRIA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditValorDiaria">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QUILOMETRAGEM:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQuilometragem">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInserirImagem">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>INSERIR IMAGEM</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImagem">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIncluir">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxModelo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MODELO:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditPlaca</tabstop>
  <tabstop>lineEditValorDiaria</tabstop>
  <tabstop>lineEditQuilometragem</tabstop>
  <tabstop>comboBoxModelo</tabstop>
  <tabstop>pushButtonInserirImagem</tabstop>
  <tabstop>pushButtonIncluir</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
